OJP/OJP_JourneySupport.xsd

```diff
@@ -63,6 +63,11 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element ref="JourneyRef"/>
+			<xs:element name="PublishedJourneyName" type="xs:normalizedString" minOccurs="0">
```

Comment:
What do you mean bt "published" ? why not just "JourneyName" ?

Reply:
changed to PublicCode. Probably better as you suggested.